scripts/darwin.sh

```diff
@@ -60,8 +60,11 @@ for disk in $DISKS; do
   else
     echo "mountpoints:"
     echo "$mountpoints" | while read -r mountpoint ; do
+      volume="$(echo "$mount_output" | perl -n -e'm{^('"${disk}"'(s[0-9]+)?) on (.*) \(.*\)$} && print "$1\n"')"
```

Comment:
Ah, good point! I'd forgotten that #168 only removes one usage of `perl`, not all usages of perl ;-)